src/v2/components/header/Header.tsx

```diff
@@ -1,26 +1,22 @@
-import { useContext } from 'react';
-import { StringsContext } from '../providers/StringsProvider/StringsContext';
 import styled, { css } from 'styled-components';
 import { H2 } from '../design/text/TextComponent';
 import { HeaderLogo } from '../HeaderLogo';
 
 export type HeaderProps = {
-	text?: string;
+	title?: string;
 	onClick?: any;
 	children?: JSX.Element;
 	showLogo?: boolean;
 };
-export const Header = ({ onClick, children, showLogo = true }: HeaderProps): JSX.Element => {
-	const { strings } = useContext(StringsContext);
-	const { HEADER_TEXT } = strings;
+export const Header = ({ onClick, children, showLogo = true, title }: HeaderProps): JSX.Element => {
 	return (
 		<HeaderWrapperContainer>
 			<HeaderContainer>
 				<StyledHeaderLogo>
 					<HeaderLogo show={showLogo} />
 				</StyledHeaderLogo>
 				<StyledHeader>
-					<StyledH2 onClick={onClick}>{HEADER_TEXT}</StyledH2>
+					<StyledH2 onClick={onClick}>{title}</StyledH2>
```

Comment:
Let's keep the HEADER_TEXT optional so that we don't need to pass it from every other component.
```suggestion
					<StyledH2 onClick={onClick}>{title || HEADER_TEXT}</StyledH2>
```